### PR TITLE
[ivy] counsel search commands use spacemacs--ivy-file-actions

### PR DESCRIPTION
**Pull request by thanhvg**
`spacemacs/counsel-search`,` counsel-rg`, `counsel-ag` and `counsel-find-file` now share a common set of actions defined in `spacemacs--ivy-file-actions`.

Updated ivy layer doc.

### CODE REVIEW

File: layers/+completion/ivy/README.org [hunk 1]
```diff
@@ -140,3 +141,37 @@ are found in [[https://oremacs.com/swiper/#minibuffer-key-bindings][the Hydra se
...
 | ~SPC h d F~ | =counsel-describe-face= |
+
+** Search files with ivy
+=ripgrep= is recommended and =Spacemacs= will pick it up as the default seach
+app if found. To pass parameters to =ripgrep= use double dash then everything
+after it is treated as parameters for search app.
+
+For example:
+#+begin_src 
+phrase I want to search -- -tlisp
```
Comment (lebensterben) [inline layers/+completion/ivy/README.org:152]: I think it's -t lisp or --type lisp

Comment (thanhvg) [reply layers/+completion/ivy/README.org:152]: The space between the short option (one letter option, `-t`) and the argument is optional. This convention holds true for many popular linux commands.

**Review by smile13241324 — APPROVED** [review]
Looks good

**Comment by smile13241324** [comment]
Thank you for contributing to spacemacs 💜, I have cherry picked your commit into develop.

**Pull request closed without merging**